gaming/schema/migrations/000001.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE players (
+	playerUUID STRING(36) NOT NULL,
+	player_name STRING(64) NOT NULL,
+	email STRING(MAX) NOT NULL,
+	password_hash BYTES(60) NOT NULL,
+	created TIMESTAMP,
+	updated TIMESTAMP,
+	stats JSON,
+	account_balance NUMERIC,
+	is_logged_in BOOL,
+	last_login TIMESTAMP,
+	valid_email BOOL,
+	current_game STRING(36)
```

Comment:
Any specific reason that we don't want to add a foreign key constraint for this column?

Reply:
Ah, not really, other than the referenced table comes in later. Using migrations like this might show the history of development. Players first, games later. 

I'll check to make sure a FK is eventually added.